variables.tf

```diff
@@ -196,7 +196,7 @@ variable "cert_uri_sans" {
 variable "cert_ttl" {
   type        = string
   description = "Optional, Time-to-live (TTL) to assign to a private certificate"
-  default     = null
+  default     = "366d"
```

Comment:
A question, should this be 364d? From a security perspective, if anything has to be less than a year, I see 364d.